### ISSUE TITLE
WebRTC option different behaviour

### ISSUE THREAD

**Issue by mikhaelkh** [gorhill/uBlock#468]
OS: Win7 SP1
Browser: [Latest Firefox x64 beta](https://ftp.mozilla.org/pub/mozilla.org/firefox/releases/latest-beta/win64/en-US/) / [Latest Chromium x64 Continuous](http://chromium.woolyss.com/download/)
Firefox about:config media.peerconnection.enabled=false
https://html5test.com/ Peer To Peer WebRTC option unchecked / checked:
Firefox: 15 / 0
Chromium: 15 / 15
TL;DR: Checking WebRTC option changes Firefox score but doesn't change Chromium score.

**Comment by gorhill** [comment]
The setting is:

> Prevent WebRTC from leaking local IP addresses

Are you leaking local IP addresses when the setting is enabled? That's what the feature is about. On Firefox it requires to disable completely WebRTC, on Chromium, WebRTC will still work though with lower performance.

**Comment by mikhaelkh** [comment]
On Firefox with `media.peerconnection.enabled=true` and `Prevent WebRTC from leaking local IP address` checked my local IP address leaked.

**Comment by gorhill** [comment]
> On Firefox with media.peerconnection.enabled=true and Prevent WebRTC from leaking local IP address checked my local IP address leaked.

Of course, if you bypass uBlock by manually setting `media.peerconnection.enabled` to true, this will defeat the setting.

**Comment by mikhaelkh** [comment]
I suppose if you can't enforce `media.peerconnection.enabled` to `true`, you should uncheck `Prevent WebRTC from leaking local IP address` or set `media.peerconnection.enabled` to `true` again.

**Comment by 5t3f4n** [comment]
I encountered an issue related to this when I first installed the uBlock update. I already had `media.peerconnection.enabled` set to `false`. The checkbox in uBlock did not reflect this, and toggling the checkbox had no effect on the setting. Re-enabling uBlock or restarting Firefox didn't help either.

Had to uninstall uBlock, reset `media.peerconnection.enabled`, re-install uBlock for it to work as expected. This was on uBlock 0.9.9.3 and Firefox 38.0.5 I believe.
